docs/docs/mlflow-3/index.mdx

```diff
@@ -36,6 +36,19 @@ When developing a new version of a GenAI agent or application, you can group all
 This enables rich, comprehensive comparisons across versions.
 Similarly, every deep learning checkpoint is stored as a <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> with its own metrics and parameters, simplifying the process of identifying the best checkpoint for deployment or continued training.
 
+### Manul control over trace grouping
+
+To manually control trace grouping, you can use <APILink fn="mlflow.set_active_model">mlflow.set_active_model</APILink> API to **set an active LoggedModel in the current context**.
```

Comment:
```suggestion
To manually control trace grouping, you can use <APILink fn="mlflow.set_active_model">mlflow.set_active_model</APILink> API to **set an active `LoggedModel` in the current context**.
```

Reply:
same comment to other LoggedModel.

Reply:
Actually I think that doesn't render pretty... We've highlighted LoggedModel in above section with a link to the LoggedModel entity, do we need to wrap it everywhere?

Reply:
Ok, I don't have a strong preference. Let's leave them as is.